birdcage_frontend/templates/annual_report.html

```diff
@@ -159,7 +161,7 @@ <h3 id="report-summary">Total Detections: 0 | Unique Species: 0</h3>
                         //console.log(`Common Name: ${commonName}, Month: ${paddedMonth}, Day: ${day + 1}, Bar Height: ${barHeight}%`);
                         return `<div class="bar" style="height: ${barHeight}%"></div>`;
                     }).join('');
-                    const totalCountDiv = monthData.total !== undefined ? `<div class="bar-total"><a href="/detections/by_common_name/${encodeURIComponent(commonName)}/${year}-${paddedMonth}-01/${year}-${paddedMonth}-${new Date(year, month, 0).getDate()}">${monthData.total}</a></div>` : '';
+                    const totalCountDiv = monthData.total !== undefined ? `<div class="bar-total"><a href="{{ script_name }}/detections/by_common_name/${encodeURIComponent(commonName)}/${year}-${paddedMonth}-01/${year}-${paddedMonth}-${new Date(year, month, 0).getDate()}">${monthData.total}</a></div>` : '';
```

Comment:
So I wasn't wild about having to "hardcode" `{{ script_name }}` into every bit of javascript that needed it, but as a relative Flask newbie I couldn't figure out a way to use `url_for` and pass in parameters that were dynamic from the  js side of things, which is presumably why you did this in the first place.

Reply:
"which is presumably why you did this in the first place."
Orrrr it just didn't occur to me.